tests/fault_tolerance/hardware/fault_injection_service/agents/gpu_fault_injector/gpu_xid_injector.py

```
@@ -153,6 +153,67 @@ def _check_privileged(self) -> bool:
+        
+        Raises:
+            FileNotFoundError: If /host/proc is not mounted
+            ValueError: If GPU ID not found
+        """
+        proc_path = "/host/proc/driver/nvidia/gpus"
+        
+        # Check if proc path exists
+        if not os.path.exists(proc_path):
+            raise FileNotFoundError(
+                f"{proc_path} not found. Ensure /host/proc is mounted in pod spec."
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)